torch/_numpy/random.py

```diff
@@ -42,10 +42,28 @@ def inner(*args, **kwds):
         if USE_NUMPY_RANDOM is False:
             return func(*args, **kwds)
         elif USE_NUMPY_RANDOM is True:
-            from numpy import random as nr
+            import numpy as _np
```

Comment:
Perhaps
```suggestion
            import numpy
```
as the `_np` thing is a bit odd.